pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -757,6 +757,30 @@ func staticRegionToZones(region string) ([]string, error) {
 			"us-west-2-lax-1b",
 			"us-west-2-wl1-sfo-wlz-1",
 		}, nil
+	case "ap-east-1":
```

Comment:
@bathina2  - I recollect using APIs to discover services(or some sort of catalog of items) from AWS using the EC2 API . Have you checked if there is an API to discover zones ?

Reply:
@bathina2 - https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/ec2-example-regions-availability-zones.html

Reply:
@onkarbhat https://github.com/kastenhq/k10/pull/10445 uses the API to get get the list and compares it with our static list. We didn't want to load the list every time, so this test runs nightly and checks to see if they are in sync

Reply:
ok, I just realized that this is a fallback mechanism . Nevermind.

Reply:
Makes sense 👍